app/controllers/pizzas_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PizzasController < ApplicationController
+  def create
+    @pizza = Pizza.new(pizza_params)
+    @pizzeria = Pizzeria.find(params[:pizzeria_id])
+    if @pizza.save
+      redirect_to pizzeria_pizza_path(pizzeria_id: @pizzeria.id, id: @pizza.id)
```

Comment:
As we learned yesterday, this will do the same thing:
```suggestion
      redirect_to [@pizzeria. @pizza]
```